bucoffea/monojet/definitions.py

```diff
@@ -977,8 +998,8 @@ def candidate_weights(weights, df, evaluator, muons, electrons, photons, cfg):
     year = extract_year(df['dataset'])
     # Muon ID and Isolation for tight and loose WP
     # Function of pT, eta (Order!)
-    weight_muons_id_tight = evaluator['muon_id_tight'](muons[df['is_tight_muon']].pt, muons[df['is_tight_muon']].abseta).prod()
```

Comment:
Same here: need to protect monojet

Reply:
Thanks, should be done in 3978b050c85843709f7131bc31775c09b950aa2c